test/e2e/common.go

```diff
@@ -61,6 +61,8 @@ const (
 	osTypeCentos           = "centos"
 	osTypeUbuntu           = "ubuntu"
 	ironicSuffix           = "-ironic"
+	// Credit for existence of this variable goes to golangci-lint.
+	True = "true"
```

Comment:
Instead of having this hack, maybe you could add something similar to this
https://github.com/metal3-io/baremetal-operator/blob/main/test/e2e/e2e_config.go#L161